apps/android/app/src/main/java/chat/simplex/app/views/chat/ComposeView.kt

```diff
@@ -118,11 +125,12 @@ fun chatItemPreview(chatItem: ChatItem): ComposePreview {
     is MsgContent.MCText -> ComposePreview.NoPreview
     is MsgContent.MCLink -> ComposePreview.CLinkPreview(linkPreview = mc.preview)
     is MsgContent.MCImage -> ComposePreview.ImagePreview(images = listOf(mc.image))
+    is MsgContent.MCVoice -> ComposePreview.VoicePreview(voice = chatItem.file?.fileName ?: "", mc.duration / 1000, true)
```

Comment:
I may understand it wrong, but this seems to imply that it's sent in milliseconds - duration should instead be passed in seconds on send